src/ini/config.c

```diff
@@ -0,0 +1,78 @@
+#include "ini.h"
+
+typedef struct /*Can be changed in the future*/
+{
+        const char *orchestrator_ip;
+        const char *ochestrator_port;
+} configurationNode;
```

Comment:
It is still not PascalCase, but rather camelCase.
Please see Michael's suggestions

Reply:
Here's the coding style for systemd:
https://github.com/systemd/systemd/blob/main/docs/CODING_STYLE.md#code-organization-and-semantics